jackson-it.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash -e
+mvn install -DskipTests=true
+for v in $(curl -s 'http://search.maven.org/solrsearch/select?q=g:%22com.fasterxml.jackson.core%22+AND+a:%22jackson-databind%22&core=gav&rows=100&wt=json' |
+           jq -r '[.response.docs | .[].v |
+                  {p: split("."), v:.} |
+                  {major:.p[0] | tonumber, minor: .p[1] | tonumber, v:.v} |
+                  select(.major >= 2 and .minor >= 4)] |
+                  sort_by(.v | split(".")) |
```

Comment:
Yeah, I skipped tonumber as I'm lazy and there's non-digits in parts of the version strings. E.g. `2.6.0-rc1`. The sorting is really just a nice to have so doesn't really need to be completely correct. It'd matter more if this was attempting to pinpoint some version that introduced a bug by bisecting etc. I wanted to use `sort -V` instead but `-V` is only in GNU sort.

I'd definitely merge a PR that implements a correct version sort in jq though ;)